etc/config/c++.amazon.properties

```diff
@@ -363,6 +363,10 @@ compiler.clang_dang.exe=/opt/compiler-explorer/clang-dang-main/bin/clang++
 compiler.clang_dang.semver=(thephd.dev)
 compiler.clang_dang.options=-std=c++2a -stdlib=libc++
 compiler.clang_dang.notification=Embed, Transparent Function Aliases, and more in this custom clang-derived playground; see <a href="https://thephd.dev/portfolio/standard" target="_blank" rel="noopener noreferrer">github<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a> for other potential proposal implementations!
+compiler.clang_widberg.exe=/opt/compiler-explorer/clang-widberg-main/bin/clang++
+compiler.clang_widberg.semver=(widberg)
+compiler.clang_widberg.options=-std=c++2a -stdlib=libc++
```

Comment:
Are these options needed? the `dang` compiler (above) needs them as it targets some very specific C++ things.